test/e2e/util_test.go

```diff
@@ -244,6 +246,52 @@ func buildSlowHTTPDPod(name, namespace string) *corev1.Pod {
 	}
 }
 
+// buildDelayConnectHTTPPod returns a pod that delays the connection and sends echo response.
+func buildDelayConnectHTTPPod(name, namespace, initImage, image string) *corev1.Pod {
+	t := true
+	return &corev1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Labels: map[string]string{
+				"app": name,
+			},
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: corev1.PodSpec{
+			InitContainers: []corev1.Container{
+				{
+					Image: initImage,
+					Name:  "init-iptables",
+					Command: []string{
+						"/bin/sh",
+						"-c",
+						"iptables -I INPUT -p tcp --dport 8080 -m conntrack --ctstate NEW -j NFQUEUE --queue-num 100",
```

Comment:
Could you add a comment here explaining why this is necessary?

I think it's creating a NFQUEUE for the `serve-delay-connect-test-server` command to work correctly no?

Reply:
Sure, will do.